teams-fluid-hello-world/src/client/helloWorldTab/HelloWorldTabConfig.tsx

```diff
@@ -0,0 +1,65 @@
+import * as React from "react";
+import { Provider, Flex, Header, Input } from "@fluentui/react-northstar";
+import { useState, useEffect, useRef } from "react";
+import { useTeams } from "msteams-react-base-component";
+import * as microsoftTeams from "@microsoft/teams-js";
+import { createContainer, containerIdString } from "./Util";
+
+/**
+ * Implementation of HelloWorldTab configuration page
+ */
+export const HelloWorldTabConfig = () => {
+
+    const [{ inTeams, theme, context }] = useTeams({});
+    const [text, setText] = useState<string>();
+    const tabName = useRef("");
+
+    const onSaveHandler = async (saveEvent: microsoftTeams.settings.SaveEvent) => {
+        const host = "https://" + window.location.host;
+        createContainer().then((containerId) => {
+            microsoftTeams.settings.setSettings({
+                contentUrl: host + "/helloWorldTab/?" + containerIdString + "=" + containerId + "&name={loginHint}&tenant={tid}&group={groupId}&theme={theme}",
+                websiteUrl: host + "/helloWorldTab/?" + containerIdString + "=" + containerId + "&name={loginHint}&tenant={tid}&group={groupId}&theme={theme}",
+                suggestedDisplayName: tabName.current,
+                removeUrl: host + "/helloWorldTab/remove.html?theme={theme}",
+                entityId: tabName.current
+            });
+        }).catch(error => console.log(error));
```

Comment:
We're in an async function and you're calling an async call. Also it's bad practice to eat and console log errors that would cripple an application. Either handle them correctly or let them bubble up.

```suggestion
        const contaienrId = await createContainer();
        microsoftTeams.settings.setSettings({
            contentUrl: host + "/helloWorldTab/?" + containerIdString + "=" + containerId + "&name={loginHint}&tenant={tid}&group={groupId}&theme={theme}",
            websiteUrl: host + "/helloWorldTab/?" + containerIdString + "=" + containerId + "&name={loginHint}&tenant={tid}&group={groupId}&theme={theme}",
            suggestedDisplayName: tabName.current,
            removeUrl: host + "/helloWorldTab/remove.html?theme={theme}",
            entityId: tabName.current
        });
```

Reply:
make sense. It was actually preventing the entire config page from working if it's unable to create the client and create the container.